src/pysight/gui/config_parser.py

```diff
@@ -54,6 +54,8 @@ def from_gui(cls, gui):
                 "bidir": gui.bidir.get(),
                 "keep_unidir": gui.keep_unidir.get(),
                 "flim": gui.flim.get(),
+                "downsampling_space": gui.downsampling_space.get(),
+                "downsampling_time": gui.downsampling_time.get(),
```

Comment:
Please change it to `flim_downsampling_x`.

Reply:
Fixed 1d94a60